### PR TITLE
Initial project setup for common module and 4 possible build targets

### PR DESCRIPTION
**Pull request by n8ebel**
Sets up a barebones configuration for a Kotlin Multiplatform project.

Includes `common` module as well as 4 targets:
- `android`
- `ios`
- `jvm` (ktor server)
- `js` (browser)

The updated README includes deployment info to test out the current output for each build target.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,14 @@
 */.DS_Store
 .DS_Store
+build/
```
Comment (n8ebel) [inline .gitignore:3]: making sure the gitignore file(s) are setup properly is definitely something I'd love another set of eyes on.  Things seem to working well now, but they can probably be cleaned up.

File: .idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (n8ebel) [inline .idea/gradle.xml:1]: On a clean clone of this branch, IntelliJ seems to import things properly and everything builds fine.

One odd thing is that IntelliJ doesn't seem to recognize the `androidMain` and `androidTest` directories as "modules" in the same way it does the other source set directories.  I'm not sure if this is something that could be configured to work differently, or if it's just how IntelliJ/Gradle understands the Android sources 🤔   I might try to compare the configuration to some other open source examples, but if you have any ideas I'm very open.  It's kind of driving me nuts 😉

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+- `cd iosapp`
+- `pod install`
+- Open `iosapp.xcworkspace` with XCode
+- Build and deploy app to simulator
+
+## Backend
+- Deploy locally with `./gradlew :backend:run`
+- Console output should include `Responding at <local browser address>`
+
+## Browser
```
Comment (n8ebel) [inline README.md:25]: This target is probably fairly unlikely to really ever manifest into anything, but I'm curious to play around with Kotlin/JS as we work on some of this other stuff.

File: common/build.gradle [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+group 'com.conferenceplatform'
+version '0.0.1' // CocoaPods requires the podspec to have a version.
+
+repositories {
+    mavenCentral()
+}
+
+android {
+    compileSdkVersion(29)
```
Comment (n8ebel) [inline common/build.gradle:15]: was planning to add and consolidate constants in a followup PR for things like kotlin version, sdk version, dependencies, etc

File: .idea/gradle.xml (hunk 2 above)
Comment (rharter) [reply .idea/gradle.xml:1]: I can take a look, but nothing jumps out at me, off the top of my head.

**Review by rharter — APPROVED** [review]
Looks good, man!

**Pull request merged**